packages/node_modules/@webex/plugin-meetings/src/locus-info/controlsUtils.js

```diff
@@ -0,0 +1,34 @@
+const controlsUtils = {};
+
+/**
+ * parse the relevant host values that we care about: id
+ * @param {Object} controls
```

Comment:
Can we add a `@typedef` for the controls object's shape?
Then we can do 
```suggestion
 * @param {ControlsObject} controls
```

Reply:
It does not have a consistent shape, it is different depending on what happens in the meeting, but I will update the shape for what we use as of now

Reply:
I see in `getId` you are calling it `LocusControls`. Let's just use that for now:
```suggestion
 * @param {LocusControls} controls
```